dao/src/integ-test/java/uk/gov/moj/sdt/dao/IndividualRequestDaoTest.java

```diff
@@ -114,111 +110,103 @@ public void setUp() {
-        Assert.assertEquals(individualRequest.getCustomerRequestReference(), customerRequestReference);
-
+        assertNotNull(individualRequest);
+        assertEquals(individualRequest.getCustomerRequestReference(), customerRequestReference);
     }
 
     /**
      * Test with an individual request past 90 days ago and use an old customer reference.
      */
     @Test
-    public void testGetIndividualRequestPastRetention() {
+    void testGetIndividualRequestPastRetention() {
         final String customerRequestReference = "customer request reference 1";
 
-        createIndividualRequest(customerRequestReference, LocalDateTime.now().plusDays((dataRetentionPeriod + 1) * -1));
+        createIndividualRequest(customerRequestReference, LocalDateTime.now().plusDays((dataRetentionPeriod + 1) * -1L));
         IIndividualRequest individualRequest = individualRequestDao.getIndividualRequest(bulkCustomer, customerRequestReference, dataRetentionPeriod);
-        Assert.assertNull(individualRequest);
-
+        assertNull(individualRequest);
     }
 
     /**
      * Test stale individual request updated 1 day ago.
      */
     @Test
-    public void testStaleIndividualRequestWithDeadLetter() {
+    void testStaleIndividualRequestWithDeadLetter() {
         // Get requests that have been updated more than 12 hrs ago.
         final List<IIndividualRequest> individualRequests = individualRequestDao.getStaleIndividualRequests(720);
 
-        Assert.assertNotNull(individualRequests);
-
-        Assert.assertEquals("The individual requests size is not correct", 2, individualRequests.size());
-
+        assertNotNull(individualRequests);
+        assertEquals(2, individualRequests.size(), "The individual requests size is not correct");
     }
 
     /**
      * Test get individual request not found.
      */
     @Test
-    public void testGetBulkSubmissionNotFound() {
-
+    void testGetBulkSubmissionNotFound() {
         final IIndividualRequest individualRequest =
                 individualRequestDao.getIndividualRequest(bulkCustomer, "NOT_FOUND", dataRetentionPeriod);
 
-        Assert.assertNull(individualRequest);
-
+        assertNull(individualRequest);
     }
 
     /**
      * Test where pending individual requests are found.
      */
     @Test
-    public void getPendingIndividualRequestsFound() {
+    void getPendingIndividualRequestsFound() {
         final List<IIndividualRequest> individualRequests = individualRequestDao.getPendingIndividualRequests(3);
 
-        Assert.assertNotNull(individualRequests);
-        Assert.assertTrue("Pending individual requests found", individualRequests.size() > 0);
-        Assert.assertEquals("The pending requests size is not correct", 1, individualRequests.size());
+        assertNotNull(individualRequests);
+        assertTrue(individualRequests.isEmpty(), "Pending individual requests found");
+        assertEquals(1, individualRequests.size(), "The pending requests size is not correct");
```

Comment:
Not sure if individualRequests.isEmpty() is doing what's expected.  The original code checked if individualRequests.size() > 0, i.e. there were some entries in the list.  It now checks if isEmpty() is true, which implies the list is empty, followed by checking that the size of the list is 1, which implies the list is not empty!  May need to revert back to original code.

Reply:
Corrected to !isEmpty()